.pre-commit-config.yaml

```diff
@@ -25,4 +27,3 @@ repos:
           - mdformat-black
           - mdformat-shfmt
           - mdformat-config
-          - mdformat-toc
```

Comment:
Keeping this is essential, else we would not autogenerate the TOC in the README

Reply:
okay, what do we have to change so that the emojis do not get removed from the TOC? Add them into the headings?